gips/0085-horizon-arbitration-charter.md

```diff
@@ -0,0 +1,209 @@
+
+In addition to the published decision, the Arbitrator must post a post-mortem in the dispute's forum thread for disputes that are settled as a Draw due to some malfunction or shortcoming of the software. This should include the specific root causes of the fault and what actions are recommended to prevent such software-related faults in the future. The Arbitrator may coordinate with core developers of the protocol to produce this post-mortem.
+
+## 8. Indexing and query faults due to software malfunctions
+
+Incorrect Proofs of Indexing or query Attestations may be produced due to software malfunctions in the Graph Node software or the Indexer's blockchain client. We will refer to these malfunctions collectively as *determinism* *bugs*.
+
+The Arbitrator is encouraged to resolve disputes as a Draw if they believe an incorrect POI or Attestation to be the result of a determinism bug.
+
+## 9. Double Jeopardy
+
+Indexers should only be slashed once for a given indexing or query fault. Any duplicate disputes for the same disputable element must be settled as a Draw. In the event that multiple Fishermen raise simultaneous disputes for the same disputable elements, only the first Fisherman to log a dispute will be considered. The remaining disputes will be settled as a Draw.
+
+A disputable element is:
+
+- For query disputes: a query attestation.
+- For indexing disputes: a POI submission (the POI plus the block number when it was submitted onchain by the indexer). Note that a POI can be challenged multiple times if it’s repeatedly being re-submitted.
+
+In the event a dispute is raised where the block number specified by the Fisherman does not include the associated POI submission the recommendation is to resolve the dispute as a Draw.
+
+## 10. Statute of Limitations
+
+Indexers should not be slashed for faults that occurred more than two thawing periods prior unless two thawing periods is less than seven (7) days. For example, if the thawing period is set to 28 epochs, then an Indexer should not be slashed for any fault that occurred more than 56 epochs prior. Note that for the **Subgraph Service** the thawing period is defined as equal to the `disputePeriod` global variable in the `DisputeManager` contract. The Arbitrator must decide any such dispute that is outside this statute of limitations as a Draw.
+
+If a POI or query attestation discrepancy is raised for dispute, leveraging historical Indexer activity beyond the statute of limitation to assist in making a determination by the arbitration council is still acceptable.
+
+## 11. Data Availability
+
+In order for an Arbitrator to settle a dispute correctly, they must have access to the following data, which is not stored on-chain:
+
+- The subgraph manifest, schema and mappings.
+- The query body (for query disputes).
```

Comment:
Nit-picking a bit here, but this will not be accessible by IPFS. This will have to be provided by the fishermen in the forum.

Reply:
fixed, thanks